src/task_7.py

```diff
@@ -0,0 +1,105 @@
+"""#Написать 12 функций по переводу:
+#Дюймы в сантиметры
+#Сантиметры в дюймы
+#Мили в километры
+#Километры в мили
+#Фунты в килограммы
+#Килограммы в фунты
+#Унции в граммы
+#Граммы в унции
+#Галлон в литры
+#Литры в галлоны
+#Пинты в литры
+#Литры в пинты
+#Примечание: функция принимает на вход число и
+возвращает конвертированное число"""
+
+
+def perevod(y, x):
+    dict = {
+        1: duim_sm,
+        2: sm_duim,
+        3: mile_km,
+        4: km_mile,
+        5: funt_kg,
+        6: kg_funt,
+        7: unc_gr,
+        8: gr_unc,
+        9: gl_litr,
+        10: litr_gl,
+        11: litr_pint,
+        12: pint_litr,
+    }
+    while True:
+        print("Введите номер операции:")
+        y = int(input())
+        if y == 0:
+            print("Конец операции")
+            break
+        else:
+            print("Введите число для перевода:")
+            x = float(input())
+            print(dict[y](x))
+
+
+def duim_sm(x):
```

Comment:
add please typing and docstring every where
```suggestion
def duim_sm(x: float) -> float:
"""Перевод дюймоф в сантиметры"""
```